Make.versions

```diff
@@ -0,0 +1,47 @@
+#
+# A release branch requires updating the following two variables at the bottom of this file:
+#
+# IOS_PACKAGE_VERSION (major/minor/revision #)
+# MAC_PACKAGE_VERSION (major/minor/revision #)
+#
+# Update version numbers on master as well, to the next version
+#
+
+#
+# ** Version numbers **
+#
+# Versions are defined as: Major.Minor.Revison.Build
+#
+# Major/minor (first/second numbers - max 2 digits each):
+# - Bump for major/minor releases.
```

Comment:
Might want to explain that for the minor version, even is for release and odd is for preview?

Reply:
good idea, but that's needed in master (not in the backport) so it will need a different PR :)